.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  arrowParens: 'always',
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
If you have a Prettier plugin for your editor, this will automatically format JS to match the project's eslint settings.